internal/commands/push.go

```diff
@@ -127,7 +129,11 @@ func runPush(dockerCli command.Cli, name string, opts pushOptions) error {
 		return errors.Wrapf(err, "fixing up %q for push", retag.cnabRef)
 	}
 	// push bundle manifest
-	descriptor, err := remotes.Push(context.Background(), bndl, retag.cnabRef, resolverConfig.Resolver, true, withAppAnnotations)
+	logger := logrus.New()
```

Comment:
nit: extract those 4 lines to a sub-function, like `ctx := newLogContext()`?

Reply:
In the same file or you think it's better to create a `utils.go` in the same repo?

Reply:
Same file is fine 👍